packages/checkbox/checkbox.scss

```diff
@@ -0,0 +1,89 @@
+@import "~@fremtind/jkl-core/build/scss/variables/colors";
+@import "~@fremtind/jkl-core/build/scss/functions";
+
+$checkbox-size: 1.5rem;
```

Comment:
There are a few other values that would be nice to parametrize as well (colors, border-widths)

Reply:
Aye. Fixed.